tests/test_cobs.cc

```diff
@@ -0,0 +1,146 @@
+#include "cobs.h"
+#include <cstring>
+
+/*
+Throw exceptions so we can catch them with gdb (catch catch).
+*/
+void throw_assert(bool condition, const char * const remark) {
+    if (condition == false) {
+        throw remark;
+    }
+}
+
+/*
+Fill a byte buffer with a given value.
+*/
+void fill(uint8_t * const b, size_t l, uint8_t v) {
+    std::memset(b, v, l);
+}
+
+/*
+Check that a byte buffer contains only a given value.
+*/
+void mem_is(uint8_t * const b, size_t l, uint8_t v) {
```

Comment:
Renaming this function to `assert_mem_is` would help readability and remind someone else (me) that this function contains an assert.